components/Shared/Settings.js

```diff
@@ -100,13 +100,13 @@ const Settings = ({ settingsIsOpen, closeSettingsModal }) => {
                   </button>
                 </span>
 
-                <div className="mt-4">
+                {/* <div className="mt-4">
```

Comment:
Hi @ankitach23, thanks for the fix, code looks good. Just wondering if we should remove the commented code if that's not needed ? 
Cheers!

Reply:
Thanks ,yes we can also remove it.